src/nats.h

```diff
@@ -3099,6 +3099,21 @@ natsOptions_DisableNoResponders(natsOptions *opts, bool disabled);
 NATS_EXTERN natsStatus
 natsOptions_SetCustomInboxPrefix(natsOptions *opts, const char *inboxPrefix);
 
+/** \brief Sets a custom padding when allocating buffer for incoming messages
+ *
+ * By default library allocates natsMsg with payload buffer size
+ * equal to payload size. Sometimes it can be useful to add some
+ * padding to the end of the buffer which can be tweaked using
+ * this option.
+ *
+ * To clear the custom message buffer padding, call this function with 0.
+ *
```

Comment:
I would like to add a note here that says that changing the option has no effect on existing NATS connections.

Reply:
Done